modules/oc-mirror-workflows-fully-disconnected-v2.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-workflows-fully-disconnected-v2_{context}"]
+= Mirroring an image set in a fully disconnected environment
+
+In a fully disconnected environment, the mirror registry that the Openshift cluster can access is not reachable from the public internet. 
+
+.Procedure
+
+. *Mirror to disk*: Prepare an archive containing the image set for mirroring. Internet access is required.
+
+. *Manual step*: Transfer the archive to the network of the disconnected mirror registry.
+
+. *Disk to mirror*: To mirror the image set from the archive to the target disconnected registry, run oc-mirror plugin v2 from the environment that has access to the mirror registry.
+
+[id="mirror-to-disk-v2_{context}"]
+== Mirroring from mirror to disk
+
+You can use the oc-mirror plugin v2 to generate an image set and save the content to disk. The generated image set can then be transferred to the disconnected environment and mirrored to the target registry.
+
+oc-mirror plugin v2 retrieves the container images from the source specified in the image set configuration and packs them into a tar archive in a local directory.
+
+.Procedure
+
+. Mirror the images from the specified image set configuration to the disk by running the following command:
++
+[source,terminal]
+----
+$ oc mirror -c isc.yaml file://<file_path> --v2 <1>
+----
+<1> Add the required file path. 
+
+.Verification
+
+. Navigate to the `<file_path>` directory that was generated.
+. Verify that the archive files have been generated.
+
+.Next steps
+
+* Configure your cluster to use the resources generated by oc-mirror plugin v2.
+
+[id="disk-mirror-v2_{context}"]
+== Mirroring from disk to mirror
+
+. Process the image set file on the disk and mirror the contents to a target mirror registry by running the following command:
++
+[source,terminal]
+----
+$ oc mirror -c isc.yaml --from file://<file_path> docker://<mirror_registry_url> --v2 <1>
+----
+<1> Specify the URL or address of the mirror registry where the images are stored and from which they need to be deleted.
+
+.Verification
+
+. Navigate to the `working-dir` directory that was generated at `<file_path>.
+. Navigate to the `cluster-resources` directory.
```

Comment:
/s . Navigate to the `cluster-resources` directory within the `working-dir` directory that was generated at `<file_path>. 

Can we write this way?

Reply:
Applies to all instances.